Makefile

```diff
@@ -38,21 +40,19 @@ docker-down:
 	docker compose down -v --remove-orphans && docker compose rm
 
 install-katana:
-	cargo install --git https://github.com/dojoengine/dojo --locked --rev be16762 katana
+	cargo install --git https://github.com/dojoengine/dojo --locked --rev dfe390 katana
```

Comment:
is this the same rev as kakarot most recent bump?

Reply:
bumped